apps/astarte_data_updater_plant/lib/astarte_data_updater_plant/data_updater/core/trigger.ex

```diff
@@ -366,11 +250,13 @@ defmodule Astarte.DataUpdaterPlant.DataUpdater.Core.Trigger do
 
       case trigger do
         {:data_trigger, %ProtobufDataTrigger{match_path: "/*"}} ->
-          {:ok, load_trigger(new_state, trigger, target)}
+          {:ok,
+           EventsTriggersHandler.Core.load_trigger(new_state.realm, trigger, target, new_state)}
```

Comment:
See above comment, here it would be 

```suggestion
           EventsTriggersHandler.cache_simple_trigger_and_target(new_state.realm, trigger, target, new_state)}
```